docs/running-locally.md

```diff
@@ -66,16 +68,6 @@ returns `allowed`).
 
 ## Developing locally
 
-Add the following to your `/etc/hosts`:
-
-```text
-127.0.0.1 dex
-127.0.0.1 minio
-127.0.0.1 postgres
-127.0.0.1 mysql
-127.0.0.1 azurite
-```
-
```

Comment:
Should probably _move_ this block into "Requirements" (which is currently the non-devcontainer section I guess?), as _only_ the devcontainer [automatically adds hosts](https://github.com/argoproj/argo-workflows/blob/7f22085d16316bdd77bdb1060455140d9d6e1664/.devcontainer/pre-build.sh#L4).

It does feel like there's more standardization and organization that could go into the local set-up -- please feel free to add follow-up PRs! I'll probably make some myself too. Thanks for bringing attention to this!